src/components/MakeWidget/subcomponents/OrderReview/OrderReview.tsx

```diff
@@ -127,57 +127,35 @@ const OrderReview: FC<OrderReviewProps> = ({
             )}
           </ReviewListItemValue>
         </ReviewListItem>
-
-        <ReviewListItem>
-          <ReviewListItemLabel>{t("orders.listed")}:</ReviewListItemLabel>
-          <ReviewListItemValue>
-            {orderType === OrderType.publicListed
-              ? t("common.yes")
-              : t("common.no")}
-          </ReviewListItemValue>
-        </ReviewListItem>
-
         <ReviewListItem>
-          <ReviewListItemLabel>{t("orders.expiryTime")}:</ReviewListItemLabel>
+          <ReviewListItemLabel>{t("orders.expiryTime")}</ReviewListItemLabel>
           <ReviewListItemValue>{expiryTranslation}</ReviewListItemValue>
         </ReviewListItem>
 
         <ReviewListItem>
-          <ReviewListItemLabel>{t("orders.exchangeRate")}:</ReviewListItemLabel>
+          <ReviewListItemLabel>{t("orders.exchangeRate")}</ReviewListItemLabel>
           <ReviewListItemValue>{rate}</ReviewListItemValue>
         </ReviewListItem>
 
         <ReviewListItem>
           <ReviewListItemLabel>
-            {`${t("orders.protocolFee")} (${protocolFee}%)`}
+            {t("orders.protocolFee")}
             <StyledIconButton
               icon="information-circle-outline"
               onClick={onFeeButtonClick}
             />
-            :
           </ReviewListItemLabel>
-          <ReviewListItemValue>{roundedFeeAmount}</ReviewListItemValue>
+          <ReviewListItemValue>
+            {roundedFeeAmount} {justifiedSignerToken?.symbol}
+          </ReviewListItemValue>
         </ReviewListItem>
 
         <ReviewListItem>
-          <ReviewListItemLabel>
-            {t("orders.totalSpending")}:
-          </ReviewListItemLabel>
+          <ReviewListItemLabel>{t("orders.totalSpending")}</ReviewListItemLabel>
           <ReviewListItemValue>
-            {roundedSignerAmountPlusFee}
+            {roundedSignerAmountPlusFee} {justifiedSignerToken?.symbol}
           </ReviewListItemValue>
         </ReviewListItem>
-
-        {justifiedSignerToken?.address === wrappedNativeToken?.address && (
```

Comment:
Enable this again?

Reply:
Not sure. Kinda bolted in and makes the widget taller. Optimally this would be an intermediate screen that explains what needs to happen. For now would it be possible to label the button as "Wrap Required ETH" or "Wrap X ETH" where X is the required amount?

Reply:
Those won't fit properly. So "Wrap ETH" fill suffice.

Ok then we'll have a separate Review screen for both approve and wrap. Let's go for that.